<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmExportImageClass</class>
 <widget class="QDialog" name="frmExportImageClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>530</width>
    <height>360</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>530</width>
    <height>360</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>530</width>
    <height>360</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>导出图像</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #bcc2bc;</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>1</number>
   </property>
   <item row="2" column="0">
    <spacer name="verticalSpacer_4">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>5</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>6</number>
     </property>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>10</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>83</width>
         <height>25</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
       </property>
       <property name="text">
        <string>单元描述：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txtDescribe">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>320</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>30</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>10</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_8">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnExecute">
       <property name="minimumSize">
        <size>
         <width>90</width>
         <height>37</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QWidget{ background-color: rgb(85, 135, 127);}
QWidget{ color: rgb(235, 235, 235);}
QWidget{ border: 2px groove gray;border-radius: 8px;padding: 2px 4px;}
QPushButton:hover{ background-color: rgb(85, 155, 127);}
QPushButton:pressed{background-color: rgb(85, 175, 127);}</string>
       </property>
       <property name="text">
        <string>执行</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>6</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>50</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QTabWidget" name="tabWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>450</width>
       <height>200</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #dfd5cf, stop: 0.15 #acbbac, stop: 0.85 #acbbac,stop:1 #dfd5cf);</string>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>基本设置</string>
      </attribute>
      <widget class="QLineEdit" name="txtLinkImage">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>18</y>
         <width>335</width>
         <height>27</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 200);
background-color: rgba(170, 170, 127,50);</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="btnLinkImage">
       <property name="geometry">
        <rect>
         <x>448</x>
         <y>18</y>
         <width>27</width>
         <height>27</height>
        </rect>
       </property>
       <property name="toolTip">
        <string>链接图像</string>
       </property>
       <property name="styleSheet">
        <string notr="true">QWidget{ border:none;border-radius: 8px;padding: 2px 4px;}
QPushButton:hover{ background-color: rgba(85, 150, 127, 100);}
QPushButton:pressed{background-color: rgba(85, 180, 127, 160);}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="sub_res.qrc">
         <normaloff>:/resource/link.png</normaloff>:/resource/link.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>27</width>
         <height>27</height>
        </size>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>18</x>
         <y>21</y>
         <width>79</width>
         <height>19</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgba(0, 0, 0, 0);</string>
       </property>
       <property name="text">
        <string>输入图像：</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnDelLinkImage">
       <property name="geometry">
        <rect>
         <x>484</x>
         <y>18</y>
         <width>27</width>
         <height>27</height>
        </rect>
       </property>
       <property name="toolTip">
        <string>删除图像链接</string>
       </property>
       <property name="styleSheet">
        <string notr="true">QWidget{ border:none;border-radius: 8px;padding: 2px 4px;}
QPushButton:hover{ background-color: rgba(226, 142, 162, 100);}
QPushButton:pressed{background-color: rgba(226, 172, 162, 200);}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="sub_res.qrc">
         <normaloff>:/resource/del_link.png</normaloff>:/resource/del_link.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>27</width>
         <height>27</height>
        </size>
       </property>
      </widget>
      <widget class="QPushButton" name="btnStoragePath">
       <property name="geometry">
        <rect>
         <x>448</x>
         <y>78</y>
         <width>32</width>
         <height>32</height>
        </rect>
       </property>
       <property name="toolTip">
        <string>存储路径</string>
       </property>
       <property name="styleSheet">
        <string notr="true">QWidget{ border:none;background-color: rgba(0, 0, 0, 0);border-radius: 8px;padding: 2px 4px;}
QPushButton:hover{ background-color: rgba(85, 150, 127, 100);}
QPushButton:pressed{background-color: rgba(85, 180, 127, 160);}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="sub_res.qrc">
         <normaloff>:/resource/tag.png</normaloff>:/resource/tag.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>18</x>
         <y>67</y>
         <width>79</width>
         <height>19</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgba(0, 0, 0, 0);</string>
       </property>
       <property name="text">
        <string>存储路径：</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnDelLinkFileName">
       <property name="geometry">
        <rect>
         <x>484</x>
         <y>146</y>
         <width>27</width>
         <height>27</height>
        </rect>
       </property>
       <property name="toolTip">
        <string>删除文件名称链接</string>
       </property>
       <property name="styleSheet">
        <string notr="true">QWidget{ border:none;border-radius: 8px;padding: 2px 4px;}
QPushButton:hover{ background-color: rgba(226, 142, 162, 100);}
QPushButton:pressed{background-color: rgba(226, 172, 162, 200);}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="sub_res.qrc">
         <normaloff>:/resource/del_link.png</normaloff>:/resource/del_link.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>27</width>
         <height>27</height>
        </size>
       </property>
      </widget>
      <widget class="QPushButton" name="btnLinkFileName">
       <property name="geometry">
        <rect>
         <x>448</x>
         <y>146</y>
         <width>27</width>
         <height>27</height>
        </rect>
       </property>
       <property name="toolTip">
        <string>链接文件名称</string>
       </property>
       <property name="styleSheet">
        <string notr="true">QWidget{ border:none;border-radius: 8px;padding: 2px 4px;}
QPushButton:hover{ background-color: rgba(85, 150, 127, 100);}
QPushButton:pressed{background-color: rgba(85, 180, 127, 160);}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="sub_res.qrc">
         <normaloff>:/resource/link.png</normaloff>:/resource/link.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>27</width>
         <height>27</height>
        </size>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>18</x>
         <y>149</y>
         <width>79</width>
         <height>19</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgba(0, 0, 0, 0);</string>
       </property>
       <property name="text">
        <string>文件名称：</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="txtFileName">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>146</y>
         <width>335</width>
         <height>27</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="toolTip">
        <string>全局变量QString类型</string>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 200);
background-color: rgba(170, 170, 127,50);</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkUseTime">
       <property name="geometry">
        <rect>
         <x>99</x>
         <y>192</y>
         <width>197</width>
         <height>23</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QCheckBox{
    color: rgb(30, 30, 30);
	border-style: flat;
	background: transparent;
}
QCheckBox::indicator {
    width: 17px;
    height: 17px;    
}
QCheckBox::indicator::unchecked {
    image: url(:/resource/untick.png);
}
QCheckBox::indicator::checked {
    image: url(:/resource/tick.png);
};</string>
       </property>
       <property name="text">
        <string>文件名自动添加时间</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="txtStoragePath">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>64</y>
         <width>335</width>
         <height>63</height>
        </rect>
       </property>
       <property name="toolTip">
        <string/>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 200);
background-color: rgba(170, 170, 127,50);</string>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>参数设置</string>
      </attribute>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>38</x>
         <y>79</y>
         <width>123</width>
         <height>19</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border: none;
background-color: rgba(0, 0, 0, 0);</string>
       </property>
       <property name="text">
        <string>图像保存天数：</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="spinImageTime">
       <property name="geometry">
        <rect>
         <x>163</x>
         <y>75</y>
         <width>66</width>
         <height>27</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QWidget{color: rgb(85, 0, 255);background-color: rgb(180, 180, 210);}
QSpinBox::up-button{
    border:transparent;
}
QSpinBox::down-button{
    border:transparent;
}
QSpinBox::up-arrow{
    image:url(:/resource/up.png);
}
QSpinBox::down-arrow{
    image:url(:/resource/down.png);
}</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>365</number>
       </property>
       <property name="value">
        <number>30</number>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>234</x>
         <y>79</y>
         <width>23</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border: none;
background-color: rgba(0, 0, 0, 0);</string>
       </property>
       <property name="text">
        <string>天</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>38</x>
         <y>20</y>
         <width>83</width>
         <height>30</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 0);
border:none;
background-color: rgba(0, 0, 0, 0);</string>
       </property>
       <property name="text">
        <string>图像格式：</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboImageFormat">
       <property name="geometry">
        <rect>
         <x>124</x>
         <y>22</y>
         <width>77</width>
         <height>30</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>77</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QComboBox {
    border: 1px solid gray;
    border-radius: 3px;  
    min-width: 5em;
    color: rgb(0, 0, 200);
	background-color: rgba(170, 170, 127,100);
}
QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 15px; 
    border-left-width: 1px;
    border-left-color: darkgray;
    border-left-style: solid;
    border-top-right-radius: 3px;
    border-bottom-right-radius: 3px;
}

QComboBox::down-arrow {
    image: url(:/resource/down.png);
} 
QComboBox::down-arrow:on {
    top: 1px;
    left: 1px;
}</string>
       </property>
       <item>
        <property name="text">
         <string>bmp</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>jpg</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>png</string>
        </property>
       </item>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="sub_res.qrc"/>
  <include location="../ImageView/sub_res.qrc"/>
 </resources>
 <connections/>
</ui>
